src/configure/clients/azure/armRestClient.ts

```diff
@@ -10,6 +10,17 @@ export class ArmRestClient {
         this.restClient = new RestClient(azureSession.credentials);
     }
 
+    public fetchArmData(endPointUri: string, httpMethod: string, body?: string){
+        return this.sendRequest(
+            this.resourceManagerEndpointUrl + endPointUri,
+            httpMethod,
+            null,
+            body
+        ).catch( (error) => {
+            console.log(error);
```

Comment:
Are you intentionally supressing exceptions?
unnecessary space before ` (error)`

Reply:
and you can use shorter version without braces
`error => console.log(error)`

Reply:
throwing error, the intent was to catch error while testing.

Reply:
if it was just for testing, then we do not need the catch block at all as exception will anyways get caught upstream

Reply:
removed catch